tools/cli/admin_elastic_search_commands_test.go

```diff
@@ -1039,6 +1039,8 @@ func TestGenerateReport(t *testing.T) {
 				// Validate the output captured by testIOHandler
 				assert.Equal(t, tt.expectedOutput, ioHandler.outputBytes.String())
 			}
+
+			os.Remove("test-report.csv")
```

Comment:
In the previous PRs, I created a file called test-report.csv. in a test case and forgot to delete it.